src/plus/integrations/authentication/cloudIntegrationService.ts

```diff
@@ -35,17 +35,26 @@ export class CloudIntegrationService {
 
 	async getConnectionSession(
 		id: IntegrationId,
-		refresh: boolean = false,
+		deprecatedAccessTokenToRefresh?: string,
```

Comment:
I'm not sure if it's deprecated or not - the api docs do not mention that it is - perhaps just `refreshToken`?

Reply:
fixed